File: redsheet-portal/src/app/database-models/v-avoid-saying-opinion.ts
import { required, maxLength, range } from '@rx/annotations';

export class vAvoidSayingOpinion {
    constructor(vAvoidSayingOpinion?: vAvoidSayingOpinion )  {
        let properties = [ "avoidSayingOpinionId", "avoidSayingOpinionValue", "cultureCountryId",];
        for (let property of properties)
            if (vAvoidSayingOpinion && vAvoidSayingOpinion[property])
                this[property] = vAvoidSayingOpinion[property];
    }
 
    @range(1,2147483647)
	avoidSayingOpinionId : number =   undefined;
 
    @required()
    @maxLength(500)
	avoidSayingOpinionValue : string =   undefined;
 
	cultureCountryId : number =   0 ;
}

File: RedSheet.DbEntities/Models/vAvoidSayingOpinion.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vAvoidSayingOpinions", Schema = "dbo")]
	[Serializable]
	public partial class vAvoidSayingOpinion  
	{
 
	[Range(1, int.MaxValue)]
	public int AvoidSayingOpinionId { get; set; } 
 
	[Required] 
	[MaxLength(500)]
	public string AvoidSayingOpinionValue { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int CultureCountryId { get; set; } 

	public vAvoidSayingOpinion()
	{
    }

    }
}

